_posts/events/2016-11-11-iot-in-the-city-collider.md

```diff
@@ -8,7 +8,7 @@ speakers:
     - Simon Chapple
     - Ritchie Somerville
     - Eoghan Howard
-venue: design-informatics
+venue: "Design Informatics"
```

Comment:
I didn't realise it was used by one of the events. For all the other past events this looked like a broken format (many commas without field values). But I'm allowing both cases now with this last commit.